CodeStarterCamp_Week2/main.swift

```diff
@@ -6,3 +6,78 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
+import Foundation
+
+let lotto: Set<Int> = getlotto()
+
+func getlotto() -> Set<Int>{
+    
+    var lotto: Set<Int> = Set<Int>()
+    
+    repeat {
+        
+        lotto.insert(Int.random(in: (1...45)))
+        
+    } while (lotto.count < 6)
+    
+    return lotto
+    
+}
+
+
+
+func getIntersectionLottoNumber() -> [Int]{
+    
+    let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+    var lottoNumber: Set<Int> = Set<Int> (myLottoNumbers)
+    
+    lottoNumber = lottoNumber.intersection(lotto)
+    
+    var lottoIntersectionArray: [Int] = [Int](lottoNumber)
+    
+    lottoIntersectionArray = lottoIntersectionArray.sorted()
```

Comment:
```suggestion
    var lottoIntersectionArray: [Int] = lottoNumber.sorted()
```

한 줄로 작성할 수 있을 것 같습니다!